Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarInteractions.cs

```diff
@@ -60,10 +65,18 @@ void Awake()
             m_PickupableLayerMask = 1 << LayerMask.NameToLayer("Pickupable");
         }
 
+        void Update()
+        {
+            CheckForPickupsInRange();
+        }
+
         public override void OnNetworkSpawn()
         {
             base.OnNetworkSpawn();
 
+            m_PickUpIndicator = FindFirstObjectByType<PickUpIndicator>();
```

Comment:
These two fetchers should only be invoked if HasAuthority is true.

Reply:
moved down after check for authority